src/utils/make-build-info.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+from string import ascii_uppercase
+import logging
+import os
+
+import requests
+
+
+DEFAULT_ENDPOINT_URL = "https://build.betaflight.com/api/options/4.5.0-zulu"
```

Comment:
Should this be an argument?

Reply:
There's an option for that, that's going to be the default if the option is absent. Currently you can change the URL via the command line option `-u`. For example:
```shell
$ src/utils/make-build-info.py -u http://www.com/restapi
```
The full command line interface is the following:
```
usage: make-build-info.py [-h] [-u ENDPOINT_URL] [-v]

options:
  -h, --help            show this help message and exit
  -u ENDPOINT_URL, --endpoint-url ENDPOINT_URL
                        URL to build options API endpoint (default: https://build.betaflight.com/api/options/4.5.0-zulu)
  -v, --verbose
```

Reply:
Awh, sorry. Perhaps you mean it should be a mandatory argument instead of an option? It would make sense since `4.5-zulu` is just simply the current version number...

Reply:
Yes soon we hit 4.6

Reply:
Ok I made the URL argument mandatory. So if run without argument it fails with:
```
make-build-info.py: error: the following arguments are required: ENDPOINT_URL
```
I don't recommend using URL fragments, should CI infrastructure, URLs whatever change, changes to the script shall not be necessary.